### PR TITLE
now uses the knowledge whether it is front or back

### PR DESCRIPTION
**Pull request by harelon**
Signed-off-by: Harel <harel.games@gmail.com>

### CODE REVIEW

File: rcv/frame_handlers/frc_2019.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def __init__(self, configuration):
         self._points = None
         self._line_contour = None
         self._response = None
+        self._is_front = self._conf.getboolean('rcv_server', 'is_front')
```
Comment (bazulay) [inline rcv/frame_handlers/frc_2019.py]: This is a property of the camera and not of the frame, there for this should be handled in the nes56_cv_manager module.

**Comment by bazulay** [comment]
Please see comment in code
:-1:

File: rcv/frame_handlers/frc_2019.py (hunk 1 above)
Comment (harelon) [reply rcv/frame_handlers/frc_2019.py]: So what i should do is on every sending from nes56_cv_manager add the front ot back field to the sent dictionary?

**Comment by bazulay** [comment]
LGMT :+1:

**Pull request merged**